### PR TITLE
Add support for apt-get to the Azure DevTest Labs artifact system.

### PR DESCRIPTION
**Pull request by d3r3kk**
# ALM Rangers Artifact: Apt-Get #
## Package Management for Debian Based Systems ##

Exposes the natural package management system for Debian-based systems to the Azure DevTest Labs users. 

Now users can install whatever software they wish via apt-get in a way that is natural to people familiar with Debian and its derivative distributions.

Please see README.md for list of supported distributions and usage.

### CODE REVIEW

File: Artifacts/linux-apt-package/Artifactfile.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{
+    "$schema": "https://raw.githubusercontent.com/Azure/azure-devtestlab/master/schemas/2015-01-01/dtlArtifacts.json",
+    "title": "Apt-Get",
+    "description": "Install packages on a Linux Azure DevTest Lab VM via apt-get",
```
Comment (drwill-ms) [inline Artifacts/linux-apt-package/Artifactfile.json]: Same as other PR feedback, please consistently spell apt-get (casing) consistently.

File: Artifacts/linux-apt-package/Artifactfile.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+            "type": "string",
+            "displayName": "Packages",
+            "allowEmpty": false,
+            "description": "The packages to install on the VM, separated by spaces. All of the package declarations supported by apt-get are supported here as well, see http://manpages.debian.org/cgi-bin/man.cgi?query=apt-get for an in-depth look at the package specifications available."
+        },
+        "update": {
+            "type": "string",
+            "displayName": "Run Update First",
+            "defaultValue": "true",
+            "description": "Run apt-get update prior to running the full install command. Allowed values are 'true' and 'false', or blank (which will evaluate to false). Note that no additional options are passed to the update command. It is recommended for you to run update."
```
Comment (drwill-ms) [inline Artifacts/linux-apt-package/Artifactfile.json]: Suggest use allowedValues property to specify

**Comment by d3r3kk** [comment]
Update documentation based on review. Note that Apt refers to the package management system as a whole, and apt-get refers to the specific command used to affect packages on a system. Capitalization is made to reflect the proper name Apt versus the technical utility apt-get.

**Comment by d3r3kk** [comment]
This PR is also updated. I'd there anything else required?

**Pull request merged**